multimodal/src/autogluon/multimodal/models/huggingface_text.py

```diff
@@ -201,6 +202,7 @@ def forward(
             {
                 LOGITS: logits,
                 FEATURES: pooled_features,
+                RAW_FEATURES: outputs.last_hidden_state * text_masks.unsqueeze(-1),  # features without pooling
```

Comment:
Where do we use `RAW_FEATURES`? It seems unused in this PR.

Reply:
Removed.